packages/integration-karma/scripts/karma-configs/base.js

```diff
@@ -33,9 +35,9 @@ function createPattern(location, config = {}) {
     };
 }
 
-function getLwcConfig(config) {
-    const compat = Boolean(config.compat);
-    const nativeShadow = Boolean(config.nativeShadow);
+function getLwcConfig() {
+    const compat = COMPAT;
+    const nativeShadow = NATIVE_SHADOW;
```

Comment:
These two values aren't used in this configuration or any of the configurations that extend it. We can just remove them since they're environment variables now.